transport/spi/src/testFixtures/kotlin/Utils.kt

```diff
@@ -69,7 +71,9 @@ fun startReceiver(configManager: ConfigManager): LinkedBlockingQueue<Message<Orc
     }
 
     Thread {
-        MessageReceiverFactory.createReceiver(OrchestratorEndpoint, configManager, ::handler)
+        runBlocking {
+            MessageReceiverFactory.createReceiver(OrchestratorEndpoint, configManager, ::handler)
+        }
     }.start()
```

Comment:
Could this be replaced with `runBlocking(Dispatchers.Default) {}` which implicitly creates a new thread?

Reply:
I don't think that this will work, since `runBlocking` waits for the termination of the passed block. The whole action here has to be executed in parallel to the test code.